packages/openactive-integration-tests/test/helpers/flow-stages/cancel-order.js

```diff
@@ -92,6 +92,21 @@ class CancelOrderFlowStage extends FlowStage {
   static getFirstOrderItemIdFromB(bFlowStage) {
     return CancelOrderFlowStage.getFirstOrderItemId(() => bFlowStage.getOutput().httpResponse.body);
   }
+
+  /**
+   * @param {BFlowStageType} bFlowStage
+   * @param {number[]} orderItemIndexes
+   */
+  static getSpecificedOrderItemIdsFromB(bFlowStage, orderItemIndexes) {
```

Comment:
Sorry this is really tiny but by `Specificed` do you mean `Specific` or `Specified`?

Reply:
The function has been renamed